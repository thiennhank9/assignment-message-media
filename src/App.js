import React from "react";
import { Provider } from "react-redux";
import store from "./redux";
import { GifsScreen } from "./screens";

function App() {
  return (
    <Provider store={store}>
      <GifsScreen />
    </Provider>
  );
}

export default App;
