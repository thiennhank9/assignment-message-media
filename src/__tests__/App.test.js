import renderer from "react-test-renderer";
import App from "../App";

test("App", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
