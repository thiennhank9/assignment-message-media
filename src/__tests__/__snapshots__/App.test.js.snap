// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App 1`] = `
<div
  className="screen"
>
  <div
    className="mt-2 container"
  >
    <div
      className="row"
    />
  </div>
  <div
    className="d-flex justify-content-center my-3"
  >
    <div
      className="d-flex flex-grow-1 justify-content-center align-items-center"
    >
      <div
        className="spinner-grow text-secondary"
      />
      <div
        className="spinner-grow text-success"
      />
      <div
        className="spinner-grow text-danger"
      />
      <div
        className="spinner-grow text-warning"
      />
      <div
        className="spinner-grow text-info"
      />
      <div
        className="spinner-grow text-light"
      />
      <div
        className="spinner-grow text-dark"
      />
    </div>
  </div>
</div>
`;
